webapi/webapi-contactsmanager-w3c-tests/contactsmanager/ContactsManager_save_Contact_fullset.html

```diff
@@ -68,7 +68,26 @@
     preferred: true,
     value: 'msn:kangx.xu@intel.com'
   });
-  var data = new Date(1987,10,27);
+  Date.prototype.format = function(format) {
```

Comment:
Hmm, good catch.

However, seems this snippet is not your origination. At least there is a post blog (forwarded) including it:
http://blog.csdn.net/chinet_bridge/article/details/15026729

Could you please point me where you get it?

CC @kangxu @chunywang; maybe we could put this extension to a support JS file so that other Date related tests can leverage it.